### DISCUSSION TITLE
[QUESTION] Upgrade to 1.2 version to 1.5, kube version 1.27+

### DISCUSSION THREAD

**Discussion by lucianiz** [longhorn/longhorn#6903] [Q&A]
Hi everyone, 

We're stuck in a strange situation now, where the cluster is version 1.27+ , can't downgrade.
We need to upgrade from 1.2 to 1.5 but longhorn doesn't support that , only from 1.2 to 1.3 
1.3 also doesn't support klusters with version over 1.24+ 

The deployment is with the helm chart.

Can anyone provide any ideas on how to proceed?

**Comment by DamiaSan** [comment]
Maybe you should upgrade step by step, for example from your version to 1.3, then from 1.3 to 1.4 and so on until 1.5.

**Reply by lucianiz** [reply]
Yeah that's the thing it's not possible to update to 1.3 because 1.3 needs kubernetes version under 1.24 . 
So the only way is to skip 1.3 and go from 1.2 -> 1.4/1.5 and we have productive use . There is no way to downgrade the kubernetes version

**Comment by DamiaSan** [comment]
But Longhorn 1.2 needs Kubernetes version under 1.24 too, so I don't understand well your environment.

**Reply by lucianiz** [reply]
Well yes, 1.2 and 1.3 can't be installed on post 1.24 kube version but if you install it on a pre 1.24 version and then the cluster gets updated but longhorn not. Longhorn keeps working , that's what happened to us.

We had version 1.20 I think when we installed and then it was autoupdated to 1.24+ after a while, but longhorn was not. The main problem is just the helm chart blocks updates and the pod security policy removal.

**Reply by DamiaSan** [reply]
Can you downgrade Kubernetes?

**Reply by lucianiz** [reply]
No, the cloud provider that we use doesn't support that.

**Reply by c3y1huang** [reply]
Which cloud provider are you using? If you are using a cloud-provider managed Kubernetes, such as AWS EKS, consider leveraging the node group feature. This enables the launch of a set of instances running an older version of Kubernetes. You can then migrate Longhorn onto these instances and then upgrade the Longhorn version.
https://longhorn.io/docs/1.5.1/advanced-resources/support-managed-k8s-service/manage-node-group-on-eks/

**Reply by lucianiz** [reply]
We're using Exoscale, a EU based provider, they don't support downgrading and the min version is 1.25 now when creating new sks clusters.

Thank you for the suggestions, looks like we're going ahead to create  a new cluster and migrate everyone there with data restore in a new longhorn version , looks like that works as expected.
